cmd/util/flags/configflags/network.go

```diff
@@ -4,10 +4,12 @@ import "github.com/bacalhau-project/bacalhau/pkg/config/types"
 
 var NetworkFlags = []Definition{
 	{
-		FlagName:     "network",
-		ConfigPath:   types.NodeNetworkType,
-		DefaultValue: Default.Node.Network.Type,
-		Description:  `Inter-node network layer type (e.g. nats, libp2p).`,
+		FlagName:          "network",
+		ConfigPath:        types.NodeNetworkType,
+		DefaultValue:      Default.Node.Network.Type,
+		Description:       `Inter-node network layer type (e.g. nats, libp2p).`,
```

Comment:
Shouldn't we remove "libp2p" here?

Reply:
We will remove this entire flag when we remove libp2p since specifying a network will no longer be an option.